gulpfile.js

```diff
@@ -17,6 +17,10 @@ const util = require('util');
 const deepmerge = require('deepmerge');
 const esbuild = require('esbuild');
 const esbuildPlugins = require('./src/build/esbuildPlugins');
+const signale = require('signale');
+const got = require('got').default;
+const unzipper = require('unzipper');
```

Comment:
It looks like this is missing in the package.jons.

Also, I would prefer using jszip over unzipper; we recently moved to jszip due to issues and CG alerts on the seemingly-abandoned unzipper package.

Usage of jszip in vscode-test: https://github.com/microsoft/vscode-test/blob/83f719c7364c316ae1736e75ef1a505ad88461f1/lib/download.ts#L278-L294